test/ctia/task/migration/store_test.clj

```diff
@@ -248,15 +247,24 @@
                   helpers/fixture-properties:clean
                   es-helpers/fixture-properties:es-store]))
 
-(p/init!)
-(def es-props (p/get-in-global-properties [:ctia :store :es]))
-(def es-conn (connect (:default es-props)))
-(def migration-index (get-in es-props [:migration :indexname]))
+(defn es-props [get-in-config]
+  (get-in-config [:ctia :store :es]))
+
+(defn es-conn [get-in-config]
+  (connect (:default (es-props get-in-config))))
+
+(defn migration-index [get-in-config]
+  (get-in (es-props get-in-config) [:migration :indexname]))
```

Comment:
I noticed this `migration-index` function differs slightly from the one in migrate-es-stores-test:

```clojure
(defn migration-index [get-in-config]
  (get-in-config [:ctia :migration :store :es :migration :indexname]))
```

Intentional?